styles/styles.css

```diff
@@ -21,19 +21,549 @@
+    position: relative;
+}
+
+.footer-section p {
+    font-size: .85rem;
+}
+
+.footer-section .triangle-container-bottom {
+    position: relative;
+    top: 0;
+    right: -10px;
+}
+
 /* --------------- 4.3.1 Form -------------------------- */
+
+.outer-container-footer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0 1rem;
+    width: auto;
+}
+
+.inner-container-footer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+    max-width: 500px;
+    gap: 1rem;
+    padding: 4rem 0 2rem 0;
+}
+
+.form-container {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+}
+
+.form-container button {
+    margin-top: 2rem;
+    width: 10rem;
+    align-self: center;
+}
+
+.form-title {
+    font-size: .75rem;
+    font-weight: bold;
+}
+
+#email-field, #name-field, #message-field {
+    padding: .75rem;
+    border: transparent;
+    border-radius: 3px;
+    font-size: .8rem;
+}
+
+#email-field, #name-field {
+    margin-bottom: 1rem;
+}
+
 /* --------------- 4.3.2 Footer Bar -------------------- */
+
+.outer-container-footer-bar {
+    background-color: var(--general-color-black);
+    color: #FFFFFF;
+}
+
+.inner-container-footer-bar {
+    font-size: .75rem;
+    text-align: center;
+    padding: 1rem;
+}
+
 /* ------------------------------- 5. Media Queries -------------------------------- */
+
+@media screen and (min-width: 400px) {
+    html {
+        font-size: 16px;
+    }
+
+    .nav-bar {
+        justify-content: flex-end;
+        gap: 2.5rem;
+        padding-right: 2rem;
+    }
+
+    .article-container {
+        height: 275px;
+        width: 275px;
+    }
+
+    .article-container p {
+        font-size: .75rem;
+    }
+}
+
+@media screen and (min-width: 500px) {
+    .introduction::after {
+        font-size: 9rem;
+    }
+
+    .projects::after {
+        font-size: 9rem;
+        top: 0;
+    }
+}
+
+@media screen and (min-width: 700px) {
+    .introduction::after {
+        font-size: 11rem;
+    }
+
+    .projects::after {
+        font-size: 11rem;
+        top: -1rem;
+    }
+
+    .outer-container-services::after {
+        font-size: 8rem;
+        top: 0;
+    }
+}
+
+@media screen and (min-width: 768px) {
+    html {
+        font-size: 18px;
+    }
+
+    .nav-bar a {
+        font-weight: normal;
+    }
+
+    .hero-section button {
+        padding: 1rem 2rem;
+    }
+
+    .outer-container-projects {
+        gap: 3rem;
+    }
+
+    .inner-container-projects {
+        flex-wrap: nowrap;
+        max-width: 1000px;
+        gap: 4rem;
+    }
+
+    .inner-container-projects:nth-child(even) {
+        flex-direction: row-reverse;
+    }
+
+    .text-wrapper {
+        flex-basis: 35%;
+        min-width: 350px;
+        flex-shrink: 0;
+        flex-grow: 0;
+        margin-left: 1rem;
+    }
+
+    .article-container p {
+        font-size: .63rem;
```

Comment:
Controleer altijd of je rem waardes overeenkomen met hele pixels. Dit is namelijk 0,63rem * 16 = 10,08px. Dat is niet lekker voor een browser om te renderen (laat staan, schalen)

Reply:
Goede tip dankjewel!